<ui version="4.0" >
 <class>CVolumeViewerClass</class>
 <widget class="QMainWindow" name="CVolumeViewerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>33</y>
     <width>878</width>
     <height>648</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="sizeConstraint" >
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label_13" >
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Data Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDataFolder" >
          <property name="maximumSize" >
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBrowse" >
          <property name="maximumSize" >
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title" >
         <string>Zoom Out Level</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Zoom Out Level:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="comboBoxZoomOutLevel" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title" >
         <string>Volume Index</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>21</height>
            </size>
           </property>
           <property name="text" >
            <string>iWidth:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>21</height>
            </size>
           </property>
           <property name="text" >
            <string>iLength:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>21</height>
            </size>
           </property>
           <property name="text" >
            <string>iHeight:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QComboBox" name="comboBoxWidth" />
         </item>
         <item row="1" column="2" >
          <widget class="QComboBox" name="comboBoxLength" />
         </item>
         <item row="2" column="2" >
          <widget class="QComboBox" name="comboBoxHeight" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDisplayVolume" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Display the Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReset" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSaveToVTK" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Save to VTK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSaveToVol" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Save to Vol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QWidget" name="tab" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>244</width>
           <height>254</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Tab 1</string>
         </attribute>
         <widget class="QLabel" name="label_14" >
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>11</y>
            <width>85</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text" >
           <string>iso Surface Value:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditValue" >
          <property name="geometry" >
           <rect>
            <x>102</x>
            <y>11</y>
            <width>133</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2" >
         <property name="geometry" >
          <rect>
           <x>4</x>
           <y>30</y>
           <width>244</width>
           <height>282</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" >
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QVTKWidget" native="1" name="vtkWidget" >
        <property name="minimumSize" >
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>681</y>
     <width>878</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>878</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global" >QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
